src/pages/example--hello-world.mdx

```diff
@@ -274,6 +249,48 @@ async function lock(lovelace, { into, owner }): Promise<TxHash> {
 }
 ```
 
+### Publishing The Locking Contract
+
+Run the excerpt above by executing:
+
+```
+deno run --allow-net --allow-read --allow-env hello_world-lock.ts
+```
+
+The above code requires you to:
+
+- have a `BLOCKFROST_API_KEY` environment variable set. You can get one by [signing up for a Blockfrost account](https://blockfrost.io/).
```

Comment:
I would personally prefer not to force an environment variable be set. For the purpose of the tutorial it's fine to have people hardcode it. Real projects probably use .env files anyways.

Reply:
I try to avoid hard coding static values, they stink.

Reply:
Yes, but this is a tutorial meant to be as low-entry barrier as possible. Though this is slightly more in line with the Hello, World! example from the core repository.